### PR TITLE
Update: add `TreePickerSimplePureDemo` component

### PR DESCRIPTION
**Pull request by lightbringer1991**
treepicker

### CODE REVIEW

File: src/styles/adslotUi/TreePickerNav.scss [hunk 1]
```diff
@@ -3,7 +3,6 @@
 .treepickernav-component {
   border-left: $border-lighter;
   border-right: $border-lighter;
-  min-height: 66px; // Fix Chrome rendering bug when the node list is longer than one page.
```
Comment (lightbringer1991) [inline src/styles/adslotUi/TreePickerNav.scss]: I couldn't see any bug when rendering a long node list

File: src/styles/adslotUi/TreePickerSimplePure.scss [hunk 2]
```diff
@@ -12,7 +12,6 @@
   .treepickernav-component {
     background-color: $color-background;
     border-top: $border-lighter;
-    min-height: 71px; // Align the rows after this item with the other pane's rows.
```
Comment (lightbringer1991) [inline src/styles/adslotUi/TreePickerSimplePure.scss]: I don't know if this is required anymore

**Comment by omgaz** [comment]
I don't think this is the right approach. The tree picker should always have a search. It aids usability and provides navigation, I'm not sure I see the value in creating a split in the implementation of this component.

If this is for the survey modal we're only hiding it so we can merge and add as a phase 2, in the time to write this the search probably could have been implemented :/

**Comment by omgaz** [comment]
As for the example (demo) that's a great improvement that can be merged.

**Comment by lightbringer1991** [comment]
@omgaz ok I'll remove the props, but will keep the demo so we have a better example for the picker

**Comment by omgaz** [comment]
LGTMerging 🍜

**Pull request merged**